cmd/serf/command/agent/invoke.go

```diff
@@ -40,7 +40,7 @@ var sanitizeTagRegexp = regexp.MustCompile(`[^A-Z0-9_]`)
 // In all events, data is passed in via stdin to facilitate piping. See
 // the various stdin functions below for more information.
 func invokeEventScript(logger *log.Logger, script string, self serf.Member, event serf.Event) error {
-	defer metrics.MeasureSince([]string{"agent", "invoke", script}, time.Now())
+	defer metrics.MeasureSinceWithLabels([]string{"agent", "invoke", script}, time.Now(), []metrics.Label{})
```

Comment:
```suggestion
	defer metrics.MeasureSinceWithLabels([]string{"agent", "invoke", script}, time.Now(), nil)
```
`nil` is a valid empty slice